joplin/js/CreateContentModal/ChooseTopicStep.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import plus from '../../static/images/plus.png';
+import minus from '../../static/images/minus.png';
+import unselected from '../../static/images/unselected.png';
+import selected from '../../static/images/selected.png';
+
+const TopicItem = ({ topic, handleTopicSelect, activeTopic }) => (
+  <li
+    onClick={handleTopicSelect.bind(this, topic.id)}
+    className="ChooseTopicStep__topic-toggle"
+  >
+    <div
+      className={
+        activeTopic === topic.id
+          ? "ChooseTopicStep__topic--selected"
+          : "ChooseTopicStep__topic--unselected"
+      }
+    ></div>
+    <span>{topic.text}</span>
+  </li>
+)
+
+class ChooseTopicStep extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      openThemeGroup: 1,
+    };
+  }
+
+  handleThemeToggle = (id, e) => {
+    // This collapses the currently selected Theme Group when it is clicked
+    if (Number(id) === this.state.openThemeGroup) {
+      this.setState({
+        openThemeGroup: 0,
+      });
+      return true;
```

Comment:
where are we using this return value?

Reply:
because if we don't return it continues on to line 38. Do you mean, we should just `return` with no value?